include/range/v3/view/remove_if.hpp

```diff
@@ -131,39 +139,43 @@ namespace ranges
             {
             private:
                 friend view_access;
-                template<typename Pred>
-                static auto bind(remove_if_fn remove_if, Pred pred)
+                template<typename Pred, typename Proj = ident>
+                static auto bind(remove_if_fn remove_if, Pred pred, Proj proj = Proj{})
```

Comment:
There are now two overloads for `bind` and `operator()`. I created a common base for `view::filter` and `view::remove_if` to avoid a complete duplications of `view::remove_if_fn`.